doc/user/depends.xml

```diff
@@ -1206,23 +1207,23 @@ Program('hello', 'hello.c', CPPPATH='.')
     <para>
 
     Apparently, the scanner does not know about the header dependency.
-    Being not a full-fledged C preprocessor, the scanner does not
+    Note being a full-fledged C preprocessor, the scanner does not
```

Comment:
Not being vs Note being?

Reply:
Sigh. Fingers don't always type what brain thought about.